cli/src/commands/explore/mod.rs

```diff
@@ -140,3 +145,63 @@ fn print_available_pallets(metadata: &Metadata) -> String {
         output
     }
 }
+
+#[cfg(test)]
+pub mod tests {
+    use super::{run, Opts};
+
+    async fn simulate_run(cli_command: &str) -> color_eyre::Result<String> {
+        let mut args = vec![
+            "explore",
+            "--file=../artifacts/polkadot_metadata_small.scale",
+        ];
+        let mut split: Vec<&str> = cli_command.split(' ').filter(|e| !e.is_empty()).collect();
+        args.append(&mut split);
+        let opts: Opts = clap::Parser::try_parse_from(args)?;
+        let mut output: Vec<u8> = Vec::new();
+        run(opts, &mut output)
+            .await
+            .map(|_| String::from_utf8(output).unwrap())
+    }
+
+    #[tokio::test]
+    async fn test_commands() {
```

Comment:
I would prefer to split this test into smaller ones because it's much harder to debug if something breaks.

Reply:
Sure, I will split it up in another PR today.